agda-categories.agda-lib

```diff
@@ -1,3 +1,3 @@
 name: agda-categories
-depend: standard-library-1.7
+depend: standard-library-2.0
```

Comment:
I made a mess trying to revert this back to 1.7, after committing it by mistake. What do?

Reply:
I can try to fix this myself, by pulling this branch.